src/components/v5/common/TableWithHeaderAndMeatballMenu/TableWithHeaderAndMeatballMenu.tsx

```diff
@@ -0,0 +1,32 @@
+import clsx from 'clsx';
+import React, { PropsWithChildren } from 'react';
+import TableWithMeatballMenu from '../TableWithMeatballMenu';
+import { TableWithHeaderAndMeatballMenuProps } from './types';
+
+const displayName = 'v5.common.TableWithHeaderAndMeatballMenu';
+
+const TableWithHeaderAndMeatballMenu = <T,>({
+  title,
+  headerClassName,
+  additionalHeaderContent,
+  children,
+  emptyContent,
+  ...rest
+}: PropsWithChildren<TableWithHeaderAndMeatballMenuProps<T>>) => (
+  <>
+    <div className={clsx(headerClassName, 'pb-4')}>
```

Comment:
I can see that the base for both tables are the same, only the table component is different. It would be better to have one component with the elements that are the same and use it for both components